### ISSUE TITLE
Devoured zombies can attack a Space Dragon from the inside

### ISSUE THREAD

**Issue by Flareguy** [space-wizards/space-station-14#23540]
## Description
If you devour a zombie as a space dragon, they will eventually revive inside of you and be able to attack you. If this happens, you can't do anything to prevent it and WILL get forcefully zombified unless the person decides to be nice.

I know #21449 exists already, but this is a much more concrete example of why this shouldn't exist. It's not difficult to perform at all; if you just devour a zombie as a space dragon, you're fucked.

**Reproduction**
1. Be a space dragon
2. Devour a zombified human
3: Have the zombified humanbite your from inside
4: God fucking damn it now I'm a zombie and my carps are going to attack me as soon as they come out of the portal.

**Comment by deltanedas** [comment]
zombie dragon should make zombie carp problem solved

**Comment by dolgovmi** [comment]
Zombies don't even regen out of death, you can just full kill them, past crit?

**Comment by Hmeister-fake** [comment]
> Zombies don't even regen out of death, you can just full kill them, past crit?

yeah they do, with every damage except heat and shock they can outheal death

**Comment by wrexbe** [comment]
I think the real lesson here is eating zombies is a terrible idea, and should be avoided.
